### ISSUE TITLE
Meteor 1.1 is breaking the update feature

### ISSUE THREAD

**Issue by anchoo2kewl** [Urigo/meteor-angular-socially#17]
While working with Meteor 1.0.5, the code to update the list of parties works. However, after upgrading to Meteor 1.1.0.2, trying to update the party name or description results in an error:

``` javascript
 Exception while simulating the effect of invoking '/parties/update' 
```

and seems to be breaking in the urigo_angular.js file when the collection tries to upgrade:

``` javascript
 if (auto) { // Deep watches the model and performs autobind.                        
        data.unregisterAutoBind = $rootScope.$watch(function(){                           
          return _.omit(data, data.$$internalProps);                                      
        }, function (newItem, oldItem) {                                                  
          if (newItem) {                                                                  
            var newItemId = newItem._id;                                                  
            if (newItemId && !_.isEmpty(newItem = _.omit(angular.copy(newItem), '_id'))) {
              collection.update({_id: newItemId}, {$set: newItem});                       
            }                                                                             
          }                                                                               
        }, true);                                                                         
      }
```

When creating save and reset buttons, the data manages to save while still throwing the same error. Without those buttons, the two way binding seems to fail and the data is not updated automatically.

**Comment by ambujpunn** [comment]
@anchoo2kewl I have the same problem as well: https://github.com/Urigo/meteor-angular-socially/pull/18
It might be a better idea to just stay at 1.0.5 for now

**Comment by ambujpunn** [comment]
Is there any update to this issue?

**Comment by Urigo** [comment]
Yes!
Fixes on angular-meteor version 0.8.4
just run

```
meteor update
```
